spec/features/user_removes_voter_from_list_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+feature 'user removes a voter from a list', %(
+As a signed up user
+I want to remove a voter from a list
+So that I can manually remove people who shouldn't be contacted
+) do
+  # Acceptance Criteria
+  # [ ] A user can delete a voter from a lists show page
+  # [ ] The user then receives a confirmation message and is returned to the
+  #     list page
+
+  scenario 'removes voter from list' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.